EIPS/eip-4762.md

```diff
@@ -186,13 +196,37 @@ If `value` is non-zero:
 (tx.target, 0, BASIC_DATA_LEAF_KEY)
 ```
 
+#### Setcode transactions
+
+For EIP-7702 transactions, for each `[[chain_id, address, nonce, y_parity, r, s], ...]` entry in the authorization list, process the additional access events:
+
+```
+(address, 0, BASIC_DATA_LEAF_KEY)
+(address, 0, CODEHASH_LEAF_KEY)
+(address, 0, CODE_OFFSET)
```

Comment:
What is `CODE_OFFSET`?

Reply:
`CODE_OFFSET` is defined in eip-6800 at `128`: it's the start of the code.

Reply:
i.e.  the first code chunk is being read

Reply:
bc1qdfs6vkttw7fj7t4m73humszxmgeq78a8we4ncf that's my address wallet

Reply:
Thanks

Reply:
😃